src/System Application/App/File System/src/Lookup/FileType.Enum.al

```diff
@@ -0,0 +1,31 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.FileSystem;
+
+/// <summary>
+/// Indicator of what type the resource is.
+/// </summary>
+enum 9452 "File Type"
```

Comment:
I know this is exactly the name and what this is representative of, and I agree with it.
But we have now seen cases of the same object name causing conflicts :( even though we have namespaces.
The name should be a little longer like `FS File Type`. Ugly but avoiding the potential future conflicts.

Reply:
Except, please don't call it FS File Type, as we're going to change the module name 😝 Let's settle on a name, then rename what needs renaming.

Reply:
I have given the enum a more specific name